packages/office-ui-fabric-react/src/components/Callout/CalloutContent.tsx

```diff
@@ -146,6 +147,12 @@ export class CalloutContent extends BaseComponent<ICalloutProps, ICalloutState>
     let getContentMaxHeight: number = this._getMaxHeight() + this.state.heightOffset!;
     let contentMaxHeight: number = calloutMaxHeight! && (calloutMaxHeight! < getContentMaxHeight) ? calloutMaxHeight! : getContentMaxHeight!;
 
+    // When using overflow-y: auto, most browsers take away room from the content div to place the scrollbar. This can result
+    // in truncated content, since there isn't as much horizontal room available. If overflow-y:scroll is set, then the scrollbar
+    // does not take away from the content width. Overflow-y: scroll always shows a scrollbar, so we need to do some measurements
+    // to check if we actually need a scrollbar.
+    const needsVerticalScrollbar: boolean = !!(this._popup && this._popup.refs.root && this._popup.refs.root.scrollHeight > this._popup.refs.root.clientHeight);
```

Comment:
hmmm.. this seems really gross..

Reply:
Any ideas? I couldn't think of a way to do this in css

Reply:
I don't like it either, but the bug is pretty bad...

Reply:
add a method or property to `IPopup` which can evaluate if a vertical scrollbar is needed, OR expose the scrollHeight/clientHeight to be read. Just don't reach into refs. "refs" is going away and really should never have been a public contract. As soon as popup is wrapped in a decorator, this won't work.